src/server/index.js

```diff
@@ -91,6 +98,11 @@ if (fs.existsSync(customConfigPath)) {
         ...customConfig.module.loaders || [],
       ],
     },
+    resolve: {
```

Comment:
I did not realized this although there is `...customConfig` in the final configuration :)

The only thing is that layout needs to be imported using an alias.

This makes me think that if in the codebase instead of importing modules using a relative path webpack was configured to make a namespace using an alias, it would be easy to override any file by overriding aliases. (I find this also handy to avoid relative paths hell in big projects)

```js
import StorybookControls from 'react-storybook/ui/controls';
import ActionLogger from 'react-storybook/ui/action_logger';
import Layout from 'react-storybook/ui/layout';
import { getSyncedStore } from 'react-storybook';
import ActionLogger from 'react-storybook/action_logger';
```

This could solve the problem in a generic way... Thoughts ?